docs/ref/forms/fields.txt

```diff
@@ -1268,6 +1279,59 @@ method::
 
         Same as :class:`ModelChoiceField.to_field_name`.
 
+    ``ModelMultipleChoiceField`` also has the attribute:
+
+    .. attribute:: iterator
+
+        Same as :class:`ModelChoiceField.iterator`.
```

Comment:
Given that folks arrive here from a search, I think repeating what we say is justified.

Reply:
This phrase was following the existing pattern of the page as used by `ModelMultipleChoiceField.queryset` and `ModelMultipleChoiceField.to_field_name`. I've updated these cases in #12174 so we consistently repeat the text rather than reference it.

Reply:
Given your comment in #12174, should I revert this back to "Same as ..."?

Reply:
I’m doing it. 🙂 Will finish my review tomorrow morning.